playground/backend/internal/setup_tools/life_cycle/life_cycle_setuper_test.go

```diff
@@ -47,16 +48,17 @@ func TestSetup(t *testing.T) {
 	}
 	defer os.RemoveAll(workingDir)
 
-	lc, err := fs_tool.NewLifeCycle(playground.Sdk_SDK_JAVA, successPipelineId, workingDir)
+	lc, err := fs_tool.NewLifeCycle(playground.Sdk_SDK_JAVA, successPipelineId, filepath.Join(workingDir, pipelinesFolder))
```

Comment:
It is not necessary. `fs_tool.NewLifeCycle` receives any directory as the last argument so it doesn't matter if it will `workingDir/executable_files` or just `workingDir`.

Reply:
The test now checks the path with executable_files, so it matter